### DISCUSSION TITLE
What is it/s a measurement of?

### DISCUSSION THREAD

**Discussion by ngold1** [vlnahp/KeyZero#8] [General]
What specifically is it/s a measurement of?

**Comment by vlnahp** [comment]
That is `iterations per second`. The number of loops passed every second.

**Reply by Huntng** [reply]
so if im getting 1 it/s does that mean 1 private key per second ?

**Reply by telejoshi** [reply]
> The number of loops passed every second.

I'm a java guy and try to learn python now. But I can't wrap my head around the number displayed as i/s:

`str(abs(n-self.prev_n)//2)`

Is this just the difference between current step (self.cur_n) and previous step (prev_n) divided by 2? How does that give i/s?

Thanks :)

**Reply by vlnahp** [reply]
> > The number of loops passed every second.
> 
> I'm a java guy and try to learn python now. But I can't wrap my head around the number displayed as i/s:
> 
> `str(abs(n-self.prev_n)//2)`
> 
> Is this just the difference between current step (self.cur_n) and previous step (prev_n) divided by 2? How does that give i/s?
> 
> Thanks :)

Hello @telejoshi.
You can find your answer in the `speed()` function. It calculates and shows the speed of the program every two seconds, so the difference between the last number and the current one is divided by 2. This function has made the program much faster.

**Reply by telejoshi** [reply]
Thanks a lot that explains it!

Just a short github question: when I download source code under "Releases" it's different from the code I can see when I open btcbf.py under "code". Why is that?

**Reply by vlnahp** [reply]
The "Release" contains the version of the repository at the time it was released. Further commits are not included.

**Reply by telejoshi** [reply]
Oh, now it seems too obvious. So the actual code is the one people are working on. Thanks.

**Reply by XWZ46999** [reply]
新版有没的教程啊
